extensions/cornerstone/src/commandsModule.js

```diff
@@ -138,9 +138,17 @@ const actions = {
     const enabledElement = getEnabledElement(viewports.activeViewportIndex);
     return enabledElement;
   },
+  getActiveViewportIndex: ({ viewports }) => {
+    return viewports.activeViewportIndex;
+  },
 };
 
 const definitions = {
+  getActiveViewportIndex: {
+    commandFn: actions.getActiveViewportIndex,
+    storeContexts: ['viewports'],
+    options: {},
+  },
```

Comment:
Commented below, but you should be able to remove this. You either have access to store contexts, or you're in the `viewer` project and can use context/redux; not sure there are many other use cases that would require something like this.

Reply:
Done! thx.